web/src/lib/api-client/apiClient.test.ts

```diff
@@ -74,11 +71,10 @@ describe("apiClient", () => {
   it("posts license status with licenseTaskId specified", async () => {
```

Comment:
pebble - you should review the test and determine if they should be renamed, rewritten, or deleted.